slapd-cli.conf

```diff
@@ -7,9 +7,15 @@ IP="*"
 PORT="389"
 SSLIP="*"
 SSLPORT="636"
+PIP="*"
+PPORT="8389"
+PSSLIP="*"
+PSSLPORT="8636"
 LDAPI_SOCKETDIR="/var/run/slapd"
 LDAPI_SOCKETURL="%2Fvar%2Frun%2Fslapd%2Fldapi"
 SLAPD_SERVICES="ldap://$IP:$PORT ldaps://$SSLIP:$SSLPORT ldapi://$LDAPI_SOCKETURL"
+# For enabling HAProxy proxy protocol
+#SLAPD_SERVICES="pldap://$PIP:$PPORT pldaps://$PSSLIP:$PSSLPORT ldapi://$LDAPI_SOCKETURL"
```

Comment:
Maybe make this example clearer ?
Here you can be confused and comment the default SLAPD_SERVICES variable to only uncomment the second one. Thus you end up disabling ldapi and standard ldap endpoints which can cause issues with the script and with slapd replication. 

Maybe something like below would be more suitable ?
`# For enabling HAProxy proxy protocol, add pldap and/or pldaps endpoint as below`

`#SLAPD_SERVICES="ldap://$IP:$PORT ldaps://$SSLIP:$SSLPORT ldapi://$LDAPI_SOCKETURL pldap://$PIP:$PPORT pldaps://$PSSLIP:$PSSLPORT `

Reply:
should be ok with last commit